app/frontend/src/components/RequestForm.vue

```diff
@@ -0,0 +1,199 @@
+              <v-icon v-on="on">help_outline</v-icon>
+            </template>
+            The Application Acronym must comply with the following format:
+            <ul>
+              <li>UPPERCASE LETTERS ONLY</li>
+              <li>Underscores may be placed between letters</li>
+              <li>Must begin and end with a letter</li>
+              <li>At least {{ fieldValidations.ACRONYM_MIN_LENGTH }} characters</li>
+              <li>
+                Examples:
+                <em>ABCD</em>,
+                <em>ABCD_WXYZ</em>
+              </li>
+            </ul>
+          </v-tooltip>
+          <v-text-field
+            placeholder="For example: 'ABC_DEF'"
+            required
+            :rules="applicationAcronymRules"
+            v-model="applicationAcronym"
+            hide-details="auto"
+            solo
+            dense
+            single-line
+            outlined
+            flat
+          >
+            <template v-slot:append-outer></template>
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <label>Comments (optional)</label>
+          <v-textarea
+            v-model="comments"
+            rows="3"
+            auto-grow
+            hide-details="auto"
+            dense
+            solo
+            outlined
+            flat
+          ></v-textarea>
+        </v-col>
+      </v-row>
+    </v-form>
+    <div class="justify-center pb-8">
+      <v-btn class="BC-Gov-PrimaryButton light float-left" text @click="cancel()">Cancel</v-btn>
+      <v-btn class="BC-Gov-PrimaryButton float-right" text @click="postRegistrationForm()">Submit</v-btn>
+    </div>
+
+    <BaseDialog v-bind:show="errorOccurred" @close-dialog="errorOccurred = false">
+      <template v-slot:icon>
+        <v-icon large color="red">cancel</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>
+          An error occurred while attempting to add your application.
+          <br />You can also add your application by sending an email to
+          <a
+            href="mailto:NR.CommonServiceShowcase@gov.bc.ca?subject=GETOK Registration for <acronym> - <idir>"
+          >NR.CommonServiceShowcase@gov.bc.ca</a>
+        </p>
+        <p>Please include your Acronym as well as your IDIR username in your email.</p>
+      </template>
+    </BaseDialog>
+
+    <BaseDialog v-bind:show="registerSuccess" @close-dialog="registerSuccess = false">
+      <template v-slot:icon>
+        <v-icon large color="green">check_circle_outline</v-icon>
+      </template>
+      <template v-slot:text>
+        <p>Your Registration was sent successfully. You will be sent an email to {{ userInfo.emailAddress }} when your application has been authorized.</p>
+      </template>
+    </BaseDialog>
+  </v-container>
+</template>
+
+<script>
+import Vue from 'vue';
+import { FieldValidations } from '@/utils/constants.js';
+
+export default {
+  name: 'RequestForm',
+  data: function() {
```

Comment:
Optional style: Could be `data() {` and would still be equivalent.

Reply:
oh yes. cheers